pkg/plugins/pluginrpc-gen/template.go

```diff
@@ -68,16 +71,21 @@ func title(s string) string {
 }
 
 var generatedTempl = template.Must(template.New("rpc_cient").Funcs(templFuncs).Parse(`
```

Comment:
Don't really understand what the linter is complaining about here...

```
pkg/plugins/pluginrpc-gen/template.go:73:86: Duplicate words (interface{},) found (dupword)
var generatedTempl = template.Must(template.New("rpc_cient").Funcs(templFuncs).Parse(`
```

Reply:
Wondering if adding `any` and `interface` (or `interface{}`?) to the list in `dupword` config would be something to consider. Adding a comment to that line in the config as well to explain why we're ignoring; https://github.com/moby/moby/blob/a3bee4156fa61760168799f2835876c926df4e60/.golangci.yml#L70-L74